internal/gen/types/typescript/typescript.go

```diff
@@ -14,9 +15,69 @@ import (
 	"github.com/supabase/cli/internal/utils"
 )
 
-var ctx = context.Background()
+func Run(ctx context.Context, useLocal bool, useLinked bool, projectId string, dbUrl string, schemas []string, fsys afero.Fs) error {
+	coalesce := func(args ...[]string) []string {
+		for _, arg := range args {
+			if len(arg) != 0 {
+				return arg
+			}
+		}
+		return []string{}
+	}
+
+	// Generating types on `projectId` and `dbUrl` should work without `supabase
+	// init` - i.e. we shouldn't try to load the config for these cases.
+
+	if projectId != "" {
+		resp, err := utils.GetSupabase().GetTypescriptTypesWithResponse(ctx, projectId, func(_ context.Context, req *http.Request) error {
+			q := req.URL.Query()
+			q.Add("included_schemas", strings.Join(coalesce(schemas, []string{"public"}), ","))
```

Comment:
Did we document this param in openapi spec? I would expect the generated request type to contain this field.

Reply:
adding the missing spec here https://github.com/supabase/infrastructure/pull/9531